docs/nextflow-postrun.ipynb

```diff
@@ -0,0 +1,268 @@
+    "However, if lower level LaminDB usage is required, it might be worthwhile writing a custom Python script.\n",
+    "\n",
+    "This guide shows how to register a Nextflow run with inputs & outputs for the example of the [nf-core/scrnaseq](https://nf-co.re/scrnaseq/latest) pipeline by running a Python script.\n",
+    "\n",
+    "The approach could be automated by deploying the script via\n",
+    "\n",
+    "1. a serverless environment trigger (e.g., AWS Lambda)\n",
+    "2. a [post-run script](https://docs.seqera.io/platform/23.4.0/launch/advanced#pre-and-post-run-scripts) on the Seqera Platform"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2",
+   "metadata": {},
+   "source": [
+    ":::{dropdown} What steps are executed by the nf-core/scrnaseq pipeline?\n",
+    "\n",
+    "![](nf_core_scrnaseq_diagram.png)\n",
+    "\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3",
+   "metadata": {
+    "tags": [
+     "hide-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "!lamin init --storage ./test-nextflow --name test-nextflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4",
+   "metadata": {},
+   "source": [
+    "## Run the pipeline\n",
+    "\n",
+    "Let's download the input data from an S3 bucket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import lamindb as ln\n",
+    "\n",
+    "input_path = ln.UPath(\"s3://lamindb-test/scrnaseq_input\")\n",
+    "input_path.download_to(\"scrnaseq_input\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6",
+   "metadata": {},
+   "source": [
+    "And run the `nf-core/scrnaseq` pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7",
+   "metadata": {
+    "tags": [
+     "hide-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# the test profile uses all downloaded input files as an input\n",
+    "!nextflow run nf-core/scrnaseq -r 4.0.0 -profile docker,test -resume --outdir scrnaseq_output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8",
+   "metadata": {},
+   "source": [
+    ":::{dropdown} What is the full run command for the test profile?\n",
+    "\n",
+    "```\n",
+    "nextflow run nf-core/scrnaseq -r 4.0.0 \\\n",
+    "    -profile docker \\\n",
+    "    -resume \\\n",
+    "    --outdir scrnaseq_output \\\n",
+    "    --input 'scrnaseq_input/samplesheet-2-0.csv' \\\n",
+    "    --skip_emptydrops \\\n",
+    "    --fasta 'https://github.com/nf-core/test-datasets/raw/scrnaseq/reference/GRCm38.p6.genome.chr19.fa' \\\n",
+    "    --gtf 'https://github.com/nf-core/test-datasets/raw/scrnaseq/reference/gencode.vM19.annotation.chr19.gtf' \\\n",
+    "    --aligner 'star' \\\n",
+    "    --protocol '10XV2' \\\n",
+    "    --max_cpus 2 \\\n",
+    "    --max_memory '6.GB' \\\n",
+    "    --max_time '6.h'\n",
```

Comment:
@Zethson did you try to run this? Running this command fails for me. It seems like you updated the `nf-core/scrnaseq` version but didn't upgrade the parameters.

Reply:
I had only looked at https://68d417e96d602a09329fdcbe--nextflow-lamin-usecases-7797.netlify.app/nextflow-postrun where we get `-[nf-core/scrnaseq] Pipeline completed successfully-` and the expected output files. Therefore I had assumed that it works as expected.